extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -94,7 +94,9 @@ When associated with a [property table](#property-tables), values of feature IDs
 
 The attribute's accessor `type` must be `"SCALAR"` and `normalized` must be false. The accessor's `componentType` is not restricted.
 
-> **Implementation Note:** since glTF accessors do not support `UNSIGNED_INT` types for 32-bit integers, `FLOAT` may be used instead allowing integer feature IDs up to 2<sup>24</sup>. For smaller ranges of feature IDs, `UNSIGNED_BYTE` or `UNSIGNED_SHORT` may be used. As with other vertex attributes, each element of a feature ID accessor must align to 4-byte boundaries.
+> **Implementation note:** since glTF accessors do not support `UNSIGNED_INT` types for 32-bit integers, `FLOAT` may be used instead allowing integer feature IDs up to 2<sup>24</sup>. For smaller ranges of feature IDs, `UNSIGNED_BYTE` or `UNSIGNED_SHORT` may be used. As with other vertex attributes, each element of a feature ID accessor must align to 4-byte boundaries.
+
+> **Implementation note:** When rasterizing a primitive with feature ID attributes, points in the interior of a triangle should be considered to belong to the feature associated with the nearest vertex.
```

Comment:
Looks mostly good, but avoid the word "rasterizing" since this implementation note could also apply to other methods like raytracing